### DISCUSSION TITLE
trice i

### DISCUSSION THREAD

**Discussion by bazylsc** [rokath/trice#511] [Q&A] [answered]
when i'm running 
trice i .
2024/11/25 13:26:56 Error in manage.go:40: func 'github.com/rokath/trice/internal/id.NewLutLI' -> invalid character 'm' looking for beginning of value

After this some macros are renumbered, some are not

**Comment by rokath** [comment, marked as answer]
The error line in the Trice code is `msg.FatalOnErr(li.fromFile(fSys, fn))`. That tells us, that your location information file, probably named `li.json` is corrupted for some reason. Can you please provide this file? The easiest way would be to delete this file and restore a new empty one: `rm li.json && touch li.json`. Intersting would be to find out why the file got corrupted. For that you should please provide more context information.
